external/fv3fit/fv3fit/keras/_models/models.py

```diff
@@ -136,98 +130,6 @@ def model(self) -> tf.keras.Model:
             raise RuntimeError("must call fit() for keras model to be available")
         return self._model
 
-    def _fit_normalization(self, X: np.ndarray, y: np.ndarray):
```

Comment:
You were able to delete so much of this! Is DenseModel used at all anymore? Or could it be deleted? If we haven't used it in a while I wouldn't be worried about backwards compatibility.

Reply:
It's not used in current experiments, so I think it's safe to delete.